test/integration/alias.test.js

```diff
@@ -28,6 +28,9 @@ beforeEach(async (t) => {
     server.register(service.api());
     const address = await server.listen();
     const folder = await fs.mkdtemp(join(os.tmpdir(), basename(__filename)));
+    await fs.mkdir(join(folder, '/map'));
+    const mapFolder = join(folder, '/map');
```

Comment:
swap these 2 lines and use mapFolder as the arg to fs.mkdir?

Reply:
Good idea!